src/urlsafe-base64-helper.js

```diff
@@ -1,32 +1,31 @@
-// Largely ported from https://github.com/RGBboy/urlsafe-base64
-
 'use strict';
 
+/**
+ * @param {Buffer} buffer
+ * @returns {string}
+ */
 function encode(buffer) {
-   return buffer.toString('base64')
-    .replace(/\+/g, '-') // Convert '+' to '-'
-    .replace(/\//g, '_') // Convert '/' to '_'
-    .replace(/=+$/, ''); // Remove ending '='
+  return buffer.toString('base64url');
 }
 
+/**
+ * @param {string} base64
+ * @returns {Buffer}
+ */
 function decode(base64) {
-   // Add removed at end '='
-  base64 += Array(5 - (base64.length % 4)).join('=');
-
-  base64 = base64
-    .replace(/-/g, '+') // Convert '-' to '+'
-    .replace(/_/g, '/'); // Convert '_' to '/'
-
-  // change from urlsafe-base64 since new Buffer() is deprecated
-  return Buffer.from(base64, 'base64');
+  return Buffer.from(base64, 'base64url');
 }
```

Comment:
Could you just remove these functions now and use `buffer.toString` and `Buffer.from` directly in the call sites?

Reply:
Done. I left `validate` in there, because there is no equivalent for that.